pkgs/applications/misc/siglo/default.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, glib
+, meson
+, ninja
+, wrapGAppsHook4
+, desktop-file-utils
+, gobject-introspection
+, gtk3
+, python3
+, python3Packages
```

Comment:
Wouldn't it be better to just use `python3` and then add packages with `python3.pkgs.*`?